org.eclipse.sisu.plexus/src/main/java/org/codehaus/plexus/PlexusTestCase.java

```diff
@@ -124,8 +123,8 @@ protected void customizeContainerConfiguration( @SuppressWarnings( "unused" ) fi
         // place-holder for tests to customize
     }
 
-    @Override
-    protected void setUp()
+    @BeforeEach
+    public void setUp()
```

Comment:
Do these have to be `public` for JUnit5?  I'm thinking if we kept them as `protected` then test classes that overrode them would still be compatible (otherwise they'd have to change their methods to be `public` to match to remove compiler warnings about making `public` methods `protected`)

That said they'd probably have to make changes anyway to remove use of any JUnit4 helpers/methods - just wondering if we can make the migration path easier for people who upgrade and then have to rewrite their tests.

Reply:
sorry I missed the comments. PR updated to make setUp and tearDown protected